src/app/ApplicationTemplate.Shared/Presentation/Authentication/ResetPasswordPageViewModel.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Chinook.DynamicMvvm;
+using Chinook.SectionsNavigation;
+using Uno.Extensions;
+
+namespace ApplicationTemplate.Presentation
+{
+	public class ResetPasswordPageViewModel : ViewModel
+	{
+		public string Password
+		{
+			get => this.Get<string>();
+			set => this.Set(value);
+		}
+
+		public bool? PasswordHasEightCharacters
+		{
+			get => this.GetFromObservable(ObservePasswordHasEightCharacters(), initialValue: null);
+			set => this.Set(value);
```

Comment:
You can usually remove the setters of properties created from Observables because you should usually not change their value manually, but rather rely only on the value from the observable.